### ISSUE TITLE
Solution to have multiple separated .js/.css/images Rails 7 without linking them in 1 big file?

### ISSUE THREAD

**Issue by Jarrodsz** [rails/rails#46152]
### Steps to reproduce
New Rails 7 application with asset pipeline and manifest.js 

The default contents of manifest with added separate .css and .js files is something like

```
//= link_tree ../images
//= link_directory ../stylesheets .css
//= link_tree ../../../vendor/javascript .js
//= link_tree ../../../vendor/javascript .js
//= link application.css
//= link site1.css
//= link site2.css
```

### Expected behavior
Of course they are all linked together in 1 single big .css file. 
But im in the need to run multiple websites from a rails application and want to have a separate .css for each of them so the expected outcome would be (for .css images and .js )

```
/assets/site1/application.css
/assets/site2/application.js
/assets/site3/images/bg.jpg
```

I could then do:

- Include the right .css in my layout ( layout for each site is different so I Link to the right .css )
- Same as above for .js
- I could do an image_tag("bg.jpg") and it would load from the correct site e.g.  /app/assets/images/site1/bg.jpg

### Actual behavior
Currently everything if of course linked. But I cannot disable the manifest.js remove its contents 
and create separated output .css. jss files.

### System configuration
Latest rails version

Ruby-3.0.4

## question

My goal is to separate images/css/js for each site.
Basically Im trying to host several websites from 1 rails app. With a custom form that is shared between them.
Without having to make a new rails project for each site I would like to somehow serve all the content from 1 rails app by operating the stylesheets. Preferable it would not be possible to query the url like /assets/site1.css or see that several websites run from 1 application.

I do a request.domain.match? "site1" etc to set some logic and load a different layout for each site
e.g. 
app/views/sites/site1/layout.haml

Anyone knows a solution to this problem? I have spending a whole day on this and found no solution online so far.

**Comment by skipkayhil** [comment]
Hey @Jarrodsz, I'm going to have to close this since this isn't a bug and we reserve the issue tracker for bugs only. However, feel free to ask on [rubyonrails-talk Discourse](https://discuss.rubyonrails.org/c/rubyonrails-talk/7) which would be a better place to ask for support with your application.
